set.mm

```diff
@@ -89722,9 +89799,10 @@ Axiom of Power Sets (unlike ~ ensymb ).  (Contributed by BTernaryTau,
     wa elin eqriv ) ABCDEZAFZBGZSCGZSDGZOZSRGTUCTUAUBSHUAUBTSIZJKUAUBTUDLMSCDPN
     Q $.
 
-  $( Natural numbers are finite sets.  (Contributed by Stefan O'Rear,
-     21-Mar-2015.) $)
-  nnfi $p |- ( A e. _om -> A e. Fin ) $=
+  $( Obsolete version of ~ nnfi as of 23-Sep-2024.  (Contributed by Stefan
+     O'Rear, 21-Mar-2015.)  (Proof modification is discouraged.)
+     (New usage is discouraged.) $)
+  nnfiOLD $p |- ( A e. _om -> A e. Fin ) $=
```

Comment:
I think nnfiOLD and enfiOLD should be kept as ALT since they are shorter, and working in ZF is more common than working in ZF \ Pow.

The comment of xxxALT begins as a rule like: `$( Alternate proof of ~ xxx `, so in this case, it could be
```
$( Alternate proof of ~ nnfi . Shorter but uses ~ ax-pow .  ...
```

On the other hand, enfiiOLD is a direct consequence of enfiOLD, so I think it can remain OLD, and be removed next year.

Reply:
I would tend to, in general, think that it is worth having both a short proof and a proof which uses fewer axioms.

In this particular case, the difference is length is small enough that I'm kind of neutral on the subject of what to do.

Reply:
In general I've been reluctant to create ALT proofs during this process because I don't want to clutter set.mm with too many of them, and because the reordering means they often won't be beside the main proof. These two proofs in particular don't seem all that important to keep to me, but if the above two concerns aren't seen as issues, then I could open another PR to turn some of the previous OLD proofs into ALT proofs. In particular I'm thinking of theorems like unfi and ssfi which I proved using induction. And if there's also consensus that nnfiOLD and enfiOLD should be made into ALT proofs, then I can update this PR to do that.

Reply:
If the ALT proofs are similar in some way, then there could just be one ALT proof standing in for all of them, like in https://us.metamath.org/mpeuni/2cnALT.html